core/db/curibio/alembic/versions/da5a93c1bb08_update_uploadtype_and_scope_schema.py

```diff
@@ -43,6 +43,11 @@ def upgrade():
             server_default="pulse3d",
         ),
     )
+    op.execute("GRANT ALL PRIVILEGES ON TABLE jobs_result TO curibio_users")
```

Comment:
why doesn't `curibio_users` need all privileges on `jobs_result`? Same for `curibio_jobs` and `customers table?

Reply:
@jperson fixed to just select privileges